scripts/astrokat-lst.py

```diff
@@ -70,6 +70,33 @@ def longformat_date(date_str):
     return date_str
 
 
+def lst2datetime(lst, date):
+    date_str = longformat_date(date)
+    utc_datetime = datetime.strptime(date_str, "%Y-%m-%d %H:%M")
+    date_lst = lst2utc(lst, Observatory().location, date=utc_datetime)
+    return ("{} {} LST corresponds to {}Z UTC"
+            .format(date, lst, date_lst))
+
+
+def date2lst(utc_datetime, observer=None):
+    if observer is None:
+        observer = Observatory().observer
+    observer.date = ephem.Date(utc_datetime)
+    return ("At {}Z MeerKAT LST will be {}"
+            .format(observer.date, observer.sidereal_time()))
+
+
+def targetlst(target_str):
+    target = ",".join(["radec target"] + target_str)
+    target = katpoint.Target(target).body
+    rise_lst = Observatory()._ephem_risetime_(target)
+    set_lst = Observatory()._ephem_settime_(target)
```

Comment:
Excellent suggestion, easiest for me is to add a public function `target_rise_and_set_times`, this will preserve the lib functionality, as well as provide the top level script with a public function instead of library functionality
```
+    def target_rise_and_set_times(self, target):
+        """Target rise and set LST times"""
+        rise_lst = self._ephem_risetime_(target)
+        set_lst = self._ephem_settime_(target)
+        return rise_lst, set_lst
```